wetterdienst/additionals/geo_location.py

```diff
@@ -1,41 +1,50 @@
 """ calculates the nearest weather station to a requested location"""
+from _datetime import datetime
 from typing import List, Union, Tuple, Optional
+
 import numpy as np
 import pandas as pd
 from scipy.spatial import cKDTree
 
-from wetterdienst.enumerations.column_names_enumeration import DWDMetaColumns
-from wetterdienst.exceptions import InvalidParameterCombination
-from wetterdienst.parse_metadata import metadata_for_climate_observations
 from wetterdienst.additionals.functions import (
     check_parameters,
     parse_enumeration_from_template,
     cast_to_list,
 )
+from wetterdienst.additionals.time_handling import parse_datetime
 from wetterdienst.data_models.coordinates import Coordinates
+from wetterdienst.enumerations.column_names_enumeration import DWDMetaColumns
 from wetterdienst.enumerations.parameter_enumeration import Parameter
 from wetterdienst.enumerations.period_type_enumeration import PeriodType
 from wetterdienst.enumerations.time_resolution_enumeration import TimeResolution
+from wetterdienst.exceptions import InvalidParameterCombination
+from wetterdienst.parse_metadata import metadata_for_climate_observations
 
 KM_EARTH_RADIUS = 6371
 
 
 def get_nearby_stations(
```

Comment:
I am finally really confused about this function. Why does it actually take in multiple longitudes/latitudes? What do these values mean to the function? Why wouldn't we use lonlat pairs? That would make a lot more sense to me!

Reply:
It was not my idea to pass multiple locations. in the end it would make the code easier to enable at least one location to ask for sites around. What do you think @gutzbenj ?

Reply:
I would prefer to limit the function to one site, as it should make things a lot easier and especially the output format a lot nicer.

Reply:
Done